src/ECER.Clients.RegistryPortal/ECER.Clients.RegistryPortal.Server/Applications/ApplicationsEndpoints.cs

```diff
@@ -84,6 +84,7 @@ public record ApplicationQueryResponse(IEnumerable<Application> Items);
 public record DraftApplication
 {
   public string? Id { get; set; }
+  public DateTime? SignedDate { get; set; }
```

Comment:
if this is just the date, you can use DateOnly

Reply:
@ytqsl Mapper was failing with DateOnly. IF Dynamics expecting DateTime would you recommend just converting in mapper DateOnly <-> DateTime?